test/test_torch.py

```diff
@@ -7614,87 +7614,6 @@ def run_test(matsize, batchdims, mat_chars):
             run_test(matsize, batchdims, mat_chars=['sym', 'sym_pd', 'sym_psd'])
             run_test(matsize, batchdims, mat_chars=['sing', 'non_sing'])
 
-    def solve_test_helper(self, A_dims, b_dims, device, dtype):
```

Comment:
thanks! good to see tests being migrated from `test_torch.py` to specific dedicated test files like `test_linalg.py`.

Reply:
for now we should just add tests in method_tests for `solve` by adding an entry here: https://github.com/pytorch/pytorch/blob/master/test/test_autograd.py#L4928-L4929 and adding complex specific tests similar to this (make sure to include 'complex' in the substring for each of the new tests so that they only run for complex dtype).